index.js

```diff
@@ -36,7 +36,7 @@ Cypress.Commands.add('mhGetAllMails', () => {
       method: 'GET',
       url: mhApiUrl('/v2/messages?limit=9999'),
     })
-    .then((response) => JSON.parse(response.body))
+    .then((response) => response.body)
```

Comment:
Maybe we could check if `response.body` is an object and return it without parsing if that is the case. Otherwise, if it is a string, we should keep parsing it and return its result. 
We would keep our support for the older Cypress versions that way.

Reply:
I added the check